terraform/modules/security-groups/main.tf

```
@@ -0,0 +1,51 @@
+resource "aws_security_group" "alb" {
+  name        = "${var.prefix}-alb-sg"
+  description = "Allow HTTP from the Internet to ALB"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = { Name = "${var.prefix}-alb-sg" }
```

```suggestion
  tags = {
    Name = "${var.prefix}-alb-sg" 
  }
```
1行でも改行するのが一般的かつ視認性が高いです！

---

他にも同様の表現の箇所が多数残っていますのでご確認お願いします！